.scalafmt.conf

```diff
@@ -1,2 +1,12 @@
-maxColumn = 100
-project.git = true
\ No newline at end of file
+align = more
```

Comment:
Not sure I am a fan of this (mostly having assignment `=`'s aligned). It's fairly unstandard I think. I would prefer `align = some`, [which is the default](https://scalameta.org/scalafmt/docs/configuration.html#align-some) to align case arrows, but not much more

Reply:
Hmm it seems i read it wrong somehow i thought none was the default, i was already doubting between some and more so ill change it to some :)